README.md

```diff
@@ -1,44 +1,147 @@
 # COMP110 Worksheet 4
 
-Please edit this README.md file with your answers to the worksheet questions.
-
 ## Question 1
 
 ### a
+|A  |B  |C  |A AND B AND NOT C|
+|F  |F  |F  |F                |
+|F  |F  |T  |F                |
+|F  |T  |F  |F                |
+|F  |T  |T  |F                |
+|T  |F  |F  |F                |
+|T  |F  |T  |F                |
+|T  |T  |F  |F                |
+|T  |T  |T  |F                |
+
 
 ### b
+|A |B |C |A AND NOT (B AND NOT C)|
+|F |F |F |F                      |
+|F |F |T |F                      |
+|F |T |F |F                      |
+|F |T |T |F                      |
+|T |F |F |T                      |
+|T |F |T |T                      |
+|T |T |F |F                      |
+|T |T |T |T                      |
+
 
 ### c
+|A |B |C |(A OR NOT B) AND (A OR C)|
+|F |F |F |F                        |
+|F |F |T |T                        |
+|F |T |F |F                        |
+|F |T |T |F                        |
+|T |F |F |T                        |
+|T |F |T |T                        |
+|T |T |F |T                        |
+|T |T |T |T                        |
 
 ### d
+|A |B |C |D |A AND NOT (B OR NOT C) AND (NOT A AND D)|
+|F |F |F |F |F                                       |
+|F |F |F |T |F                                       |
+|F |F |T |F |F                                       |
+|F |F |T |T |F                                       |
+|F |T |F |F |F                                       |
+|F |T |F |T |F                                       |
+|F |T |T |F |F                                       |
+|F |T |T |T |F                                       |
+|T |F |F |F |F                                       |
+|T |F |F |T |F                                       |
+|T |F |T |F |F                                       |
+|T |F |T |T |F                                       |
+|T |T |F |F |F                                       |
+|T |T |F |T |F                                       |
+|T |T |T |F |F                                       |
+|T |T |T |T |F                                       |
 
 ## Question 2
 
-### a
-
-### b
-
-### c
-
-### d
+https://imgur.com/7su0dsE
```

Comment:
a: the NAND gates should be AND gates
b: the lower AND gate should be a NAND
c: ✔
d: you seem to have two outputs (the top AND gate isn't wired into anything). The OR gate should be a NOR also.